### PR TITLE
DOCSP-26855 AtlasCLI 1.4.0 Release Notes

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [DOCSP-26855](https://jira.mongodb.org/browse/DOCSP-26855)
- [Stage](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-26855/atlas-cli-changelog/)
- [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6398c539dc742b91f22e56d0) (contains errors)

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -12,6 +12,23 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.4.0:
+
+{+atlas-cli+} 1.4.0
+-------------------
+
+Released 13 December 2022
```
Comment (melissamahoney-mongodb) [inline source/atlas-cli-changelog.txt:20]:
```suggestion
Released 5 December 2022
```

This actually came out awhile ago!

**Review by melissamahoney-mongodb — CHANGES_REQUESTED** [review]

**Review by melissamahoney-mongodb — APPROVED** [review]
LGTM, and the build log contains known errors

File: source/atlas-cli-changelog.txt [hunk 2]
```diff
@@ -12,6 +12,23 @@
...
+
+- Fixes the :ref:`atlas-liveMigrations-cutover` command to take the ``orgId`` instead
+  of the ``projectId``. 
+- Adds options to the :ref:`atlas-kubernetes-config-generate` command for the
+  following: 
+  
+  - Exporting |service| teams.
+  - Exporting private endpoints for serverless instances.    
+  - Exporting resources from Atlas and convert them to the
+    :atlas:`AtlasOperator </atlas-operator/>` resources.
```
Comment (gssbzn) [inline source/atlas-cli-changelog.txt:30]: FYI @igor-karpukhin

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**